src/GitVersionCore/IncrementStrategyFinder.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace GitVersion
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using VersionCalculation.BaseVersionCalculators;
+    using LibGit2Sharp;
+
+    public enum CommitMessageVersionMode
```

Comment:
Ok I get it.

Maybe name the setting `commit-message-incrementing: [enabled|disabled|merge-message-only]`

The enum can them become `CommitMessageIncrementing`

Thoughts on naming?